modules/kn-service-describe.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_serving/serverless-applications.adoc
+// * serverless/cli_reference/kn-services-ref.adoc
+
+[id="kn-service-describe_{context}"]
+= Describing serverless applications
+
+You can describe a Knative service by using the `kn` CLI `kn service describe` command.
```

Comment:
s/the `kn` CLI `kn service describe` command/the `kn service describe` command

Not sure you need the `kn` CLI part. I assume our users would know the command is CLI.

Reply:
Good call!